v3/src/components/graph/adornments/movable-line.tsx

```diff
@@ -166,16 +235,37 @@ export const MovableLine = (props: {
       .attr('class', 'movable-line-cover movable-line-middle-cover')
     newLineObject.upper = selection.append('line')
       .attr('class', 'movable-line-cover movable-line-upper-cover')
-    const equationDiv = select(kGraphClassSelector).append('div')
-      .attr('class', 'movable-line-equation-container')
+    newLineObject.handleLower = selection.append('rect')
+        .attr('class', 'movable-line-handle movable-line-lower-handle')
+    newLineObject.handleMiddle = selection.append('rect')
+        .attr('class', 'movable-line-handle movable-line-middle-handle')
+    newLineObject.handleUpper = selection.append('rect')
+        .attr('class', 'movable-line-handle movable-line-upper-handle')
+
+    // Set up the corresponding equation box
+    // Define the selector that corresponds with the adornment container
+    // TODO: Make this value more specific or find another way to target the correct element. It's currently 
+    // causing a bug when there is more than one graph present -- the equation box can be added to the wrong 
+    // graph because the selector is too generic.
```

Comment:
You can use the instance id (`useInstanceIdContext()`) to scope to a particular tile instance.